src/tribol/mesh/MfemData.cpp

```diff
@@ -696,43 +708,53 @@ std::unique_ptr<mfem::BlockOperator> MfemJacobianData::GetMfemBlockJacobian(
   // (0,1) and (1,0) are symmetric (for now using SINGLE_MORTAR with approximate tangent)
   const auto& elem_map_1 = parent_data_.GetElemMap1();
   const auto& elem_map_2 = parent_data_.GetElemMap2();
-  auto mortar_elems = method_data
```

Comment:
This stuff is a bit of a hack because `MethodData` does not exist when the rank has no mesh.

It might make more sense to create `MethodData` even if the rank doesn't have a mesh.  It would get rid of the branching here and create a more unified code path for all ranks.  The downside is you'll have to allocate things unnecessarily (as noted [here](https://github.com/LLNL/Tribol/blob/develop/src/tribol/mesh/CouplingScheme.cpp#L975-L977)), but the rank where things are being allocated to is likely going to have to wait anyway for other ranks to finish processing their data.  Let's discuss.